.travis.yml

```diff
@@ -13,3 +13,4 @@ script:
   - cmake -DCMAKE_BUILD_TYPE=Release -DDEPENDENCY_SEARCH_PREFIX=$VIRTUAL_ENV -DAUTOBUILD_TESTS=1 -DAUTOEXEC_TESTS=0 -DVIGRANUMPY_LIBRARIES="/opt/python/${TRAVIS_PYTHON_VERSION}*/lib/libpython${TRAVIS_PYTHON_VERSION}.so;/usr/lib/libboost_python.so" ..
   - make
   - make test
+  - ./test/blockwisealgorithms/test_blockwiselabeling
```

Comment:
It might be possible to keep track of the failed tests and re-run them with more verbose output. But this doesn't sound like it is easy to do.